modyn/config/schema/modyn_config_schema.yaml

```diff
@@ -99,15 +102,34 @@ properties:
                 record_size:
                   type: number
                   description: |
-                    The size of each full record in bytes (label + features) for a binary file wrapper.
+                    [BinaryFileWrapper] The size of each full record in bytes (label + features).
                 label_size:
                   type: number
                   description: |
-                    The size of the label field in bytes for a binary file wrapper.
+                    [BinaryFileWrapper] The size of the label field in bytes for a binary file wrapper.
                 byteorder:
                   type: string
                   description: |
-                    The byteorder when reading an integer from multibyte data in a binary file. Should either be "big" or "little".
+                    [BinaryFileWrapper] The byteorder when reading an integer from multibyte data in a binary file. 
+                    Should either be "big" or "little".
+                separator:
+                  type: string
+                  description: |
+                    [CsvFileWrapper] The separator used in the CSV file. The default is ";"
```

Comment:
I think usually the default for csv is `,`, isn't it (after all, csv means comma separated values :D)

Reply:
OK, for me CSV has always been with the semicolon. I investigated a bit and apparently it's my Italian origin (as decimals are separated by the comma, like `1.32` is `1,32` so typically ; is used as separator. And this is handled automatically by the operating system depending on the OS language).

Anyway, now the comma is the default.